### PR TITLE
Interface counters enhancements and bug fix

### PR DESCRIPTION
**Pull request by rvasanthm**
1. Added support for RX/TX oversize pkts and RX pkts of size between 128 to 255 octects
2.  Fixed issue in handling show interface management when state object is not present for ip addresses

### CODE REVIEW

File: src/translib/transformer/xfmr_intf.go [hunk 1]
```diff
@@ -1703,6 +1731,20 @@ var populatePortCounters PopulateIntfCounters = func (inParams XfmrParams, count
                 err = errors.New("Get Counter URI failed")
             }
         }
+    case "/openconfig-interfaces:interfaces/interface/ethernet/state/counters",
+         "/openconfig-interfaces:interfaces/interface/openconfig-if-ethernet:ethernet/state/counters":
+        for _, attr := range etherCntList {
+            uri := targetUriPath + "/" + attr
+            if ok, err := getSpecificCounterAttr(uri, &CounterData, &CounterBackUpData, counter); !ok || err != nil {
+                log.Info("Get Ethernet Counter URI failed :", uri)
+                err = errors.New("Get Ethernet Counter URI failed")
```
Comment (justinejose91) [inline src/translib/transformer/xfmr_intf.go:1740]: Do we need to return here or continue processing?

File: src/translib/transformer/xfmr_intf.go [hunk 2]
```diff
@@ -1816,8 +1860,8 @@ var populateMGMTPortCounters PopulateIntfCounters = func (inParams XfmrParams, c
     stats := strings.Fields(entry)
     log.Info(" Interface filds: ", stats)
 
-    ret := getMgmtSpecificCounterAttr(targetUriPath, stats, counter)
-    log.Info(" getMgmtCounters : ", *counter)
+    ret := getMgmtSpecificCounterAttr(targetUriPath, stats, counter_val)
+    log.Info(" getMgmtCounters : ", *counter_val)
```
Comment (justinejose91) [inline src/translib/transformer/xfmr_intf.go:1864]: counter_val can be nil right, if we try to print, it can cause panic.

**Review by justinejose91 — APPROVED** [review]

File: src/translib/transformer/xfmr_intf.go (hunk 1 above)
Comment (rvasanthm) [reply src/translib/transformer/xfmr_intf.go:1740]: This loop is for all counters, if one fails will report the log and continue filling other attributes.

File: src/translib/transformer/xfmr_intf.go (hunk 2 above)
Comment (rvasanthm) [reply src/translib/transformer/xfmr_intf.go:1864]: Counter_val is built by the caller, we are just populating attributes and dumping as part of this log.

**Pull request merged**